website/src/components/Error.tsx

```diff
@@ -0,0 +1,14 @@
+    import { Alert, AlertTitle, Box } from "@mui/material";
+
+    
+    export function Error() {
+        return (           
+            <Box display="flex" justifyContent="center" alignItems="center" minHeight="100vh">
+                <Alert variant="filled" severity="error" sx={{m: 5, p: 3, minWidth: 500}}>
+                    <AlertTitle>Error loading the page</AlertTitle>
+                    An error has occurred!
```

Comment:
I would prefer the error box showing the actual error message. We could achieve this by:
- letting the `AppState.loadingError` hold the error message, or `null` if none present
- letting this `Error` function asking for an argument, e.g. ` export function Error({ message }) {` 
- using the argument somehow in the return string `{ message }`
- providing the loading error to the component on invocation, e.g. by `<Error message={ AppState.loadingError }/>`

(The improvement of the actual error message content would be a topic for another PR.)

Reply:
![image](https://github.com/freifunkMUC/wg-access-server/assets/5702338/17428be0-07e2-4b90-a7c7-d92cfea67aa1)